app-doc/gimp-help/gimp-help-2.10.0.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 )
+inherit python-any-r1
+
+DESCRIPTION="GNU Image Manipulation Program help files"
+HOMEPAGE="https://docs.gimp.org/"
+SRC_URI="mirror://gimp/help/${P}.tar.bz2"
+
+LICENSE="FDL-1.2"
+SLOT="2"
+KEYWORDS="~alpha ~amd64 ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86"
+IUSE=""
+
+DEPEND="${PYTHON_DEPS}
+	$(python_gen_any_dep 'dev-libs/libxml2[python,${PYTHON_USEDEP}]')
+	dev-libs/libxslt
+	sys-devel/gettext
+"
+
+src_configure() {
+	econf --without-gimp
```

Comment:
Hmm, what does this do?

Reply:
This option is require to install `gimp-help` without `media-gfx/gimp` installation.
I assume that such doc package doesn't  require the package it describes.

Without it (and without installed GIMP) a configuration error is recieved:
```
checking for GIMP... no
configure: error:
** Couldn't find a GIMP-2.0 installation. If you want to skip this check and
** install the help files to the configured prefix, use the --without-gimp
** configure option. Note that GIMP will then only find the help files if it
** is installed in the same prefix. See the file 'INSTALL' for more help.
```